doc/release-notes.md

```diff
@@ -49,27 +52,223 @@ so downgrading is not recommended.
+it provides and align with industry standard terminology. The renaming only
+applies to the UI and RPCs but does not change functionality.
+
+Sporks
+------
+Several spork changes have been made to streamline code and improve system
+reliability. Activation of `SPORK_22_PS_MORE_PARTICIPANTS` in DashCore v0.16
+has rendered that spork unnecessary. The associated logic has been hardened
+and the spork removed. `SPORK_21_QUORUM_ALL_CONNECTED` logic has been split
+into two sporks, `SPORK_21_QUORUM_ALL_CONNECTED` and `SPORK_23_QUORUM_POSE`,
+so that masternode quorum connectivity and quorum Proof of Service (PoSe) can
+be controlled independently. Finally, `SPORK_2_INSTANTSEND_ENABLED` has a new
+mode (value: 1) that enables a smooth transition in case InstantSend needs to
+be disabled.
+
+Build system
+------------
+Multiple packaged in `depends` were updated. Current versions are:
+- biplist 1.0.3
+- bls-dash 1.0.1
+- boost 1.70.0
+- cmake 3.14.7
+- expat 2.2.5
+- mac alias 2.0.7
+- miniupnpc 2.0.20180203
+- qt 5.9.6
+- zeromq 4.2.3
+
+Packages libX11, libXext, xextproto and xtrans are no longer used.
+
+Minimum supported macOS version was bumped to 10.10.
+
+RPC changes
+-----------
+There are six new RPC commands which are Dash specific and seven new RPC
+commands introduced through Bitcoin backports. One previously deprecated RPC,
+estimatefee, was removed and several RPCs have been deprecated.
+
+The new RPCs are:
+- `createwallet`
+- `getaddressesbylabel`
+- `getaddressinfo`
+- `getzmqnotifications`
+- `gobject list-prepared`
+- `listlabels`
+- `masternode payments`
+- `quorum getdata`
+- `quorum verify`
+- `signrawtransactionwithkey`
+- `signrawtransactionwithwallet`
+- `unloadwallet`
+- `verifyislock`
+- `upgradetohd`
+
+The deprecated RPCs are all related to the deprecation of wallet accounts and
+will be removed in DashCore v0.18. Note that the deprecation of wallet accounts
+means that any RPCs that previously accepted an “account” parameter are
+affected — please refer to the RPC help for details about specific RPCs.
+
+The deprecated RPCs are:
+- `getaccount`
+- `getaccountaddress`
+- `getaddressbyaccount`
+- `getreceivedbyaccount`
+- `listaccounts`
+- `listreceivedbyaccount`
+- `masternode current`
+- `masternode winner`
+- `move`
+- `sendfrom`
+- `setaccount`
+
+`protx register` and `protx register_fund` rpcs now accept additional param
+`submit` which allows to produce and print ProRegTx-es without relaying them
+to the network.
+
+Also, please note that all mixing-related RPCs have been renamed to replace
+“PrivateSend” with “CoinJoin” (e.g. `setprivatesendrounds` -> `setcoinjoinrounds`).
+
+Please check `help <command>` in rpc for more information.
+
+Command-line options
+--------------------
+Changes in existing cmd-line options:
+
+New cmd-line options:
+- `-dip8params`
+- `-llmq-data-recovery`
+- `-llmq-qvvec-sync`
+- `-llmqinstantsend`
+- `-platform-user`
+- `-statsenabled`
```

Comment:
Is this the first release that includes the statoshi stuff? Might be worth mentioning in the release notes.